app/components/services/servicesController.js

```diff
@@ -72,15 +72,28 @@ function ($scope, $stateParams, $state, Service, ServiceHelper, Messages, Pagina
 
   function fetchServices() {
     $('#loadServicesSpinner').show();
-    Service.query({}, function (d) {
-      $scope.services = d.map(function (service) {
-        return new ServiceViewModel(service);
+    $q.all({
+      services: Service.query({}).$promise,
+      tasks: Task.query({filters: {'desired-state': ['running']}}).$promise,
+      nodes: Node.query({filters: {membership: ['accepted']}}).$promise,
```

Comment:
It's already filtered client-side (https://github.com/portainer/portainer/pull/614/files#diff-c6751901757a126db383af452eca289bR86). I think removing the membership query will solve the issue, I can push a new commit later this evening.